src/ZodiacCard/ZodiacCard.js

```diff
@@ -1,5 +1,13 @@
-import './ZodiacCard.css'
+import React from 'react';
+import './ZodiacCard.css';
 
-export default function ZodiacCard() {
+export default function ZodiacCard({ name, dates }) {
```

Comment:
nice job desctructuring name and date off of the zodiac object from props.

Reply:
Thank you!